### PR TITLE
Net_printer.py argument bug

### PR DESCRIPTION
**Pull request by DeepakVelmurugan**
Fixes issues mentioned in #41660

### CODE REVIEW

**Comment by DeepakVelmurugan** [comment]
Hi @malfet could you review this PR and merge if no bugs:)

File: caffe2/python/pybind_state.cc [hunk 1]
```diff
@@ -1749,7 +1749,6 @@ void addGlobalMethods(py::module& m) {
   m.def(
       "onnxifi",
       [](const py::bytes& pred_net_str,
-         const py::bytes& shapes_str,
```
Comment (malfet) [inline caffe2/python/pybind_state.cc:1752]: Why are you deleting this? And this would cause the code compilation failures.

**Review by malfet — CHANGES_REQUESTED** [review]
Please fix compilation failures and do not bundle unrelated issues  into the same PR.

**Review by DeepakVelmurugan — COMMENTED** [review]
Sorry I wasn't aware of that

File: caffe2/python/pybind_state.cc (hunk 1 above)
Comment (DeepakVelmurugan) [reply caffe2/python/pybind_state.cc]: @malfet I've made the necessary changes.

**Comment by DeepakVelmurugan** [comment]
@malfet could you please merge this PR?

**Pull request closed without merging**